compio-runtime/src/runtime/mod.rs

```diff
@@ -241,6 +244,26 @@ impl Runtime {
         }
     }
 
+    /// Submit an operation to the runtime.
+    ///
+    /// The difference between [`Runtime::submit`] is this method will return
+    /// the flags
+    ///
+    /// You only need this when authoring your own [`OpCode`].
+    pub fn submit_with_flags<T: OpCode + 'static>(
```

Comment:
I think the `submit` method could be implemented upon this one.

Reply:
done